example-documents/README.md

```diff
@@ -1,3 +1,50 @@
-this folder contains example documents 
-from past teams that we consider are of
-high quality. 
\ No newline at end of file
+This folder contains example documents from past teams that we consider are of high quality. 
+Use them as a rough guideline, not as a template.
+
+As a general note: try to keep your documents **consise**. That is: short but descriptive.
+The sad truth of the world is that people do not like to read large documents, and if your document is not being read you are wasting valuable development time ;).
+
+## Requirements Document
+The requirements document should be self contained and should  also make sense to somebody who is not part of the team. Thus it should have a high-level introduction that explains the importance and the scope of the project. But the most important role of this document, is that it will be the evolving shared understanding between you and the customer. Thus, the more details it contains the more can you be sure that you are on the same page. And this is why we ask you to update it after every meeting with the customer. 
+
```

Comment:
Imagine that you are writing the document for a team new member who will join the project halfway through. Or even for a team that will have to continue with the project once you are finished. Will they understand what needs to be done?